IRremoteESP8266.h

```diff
@@ -32,6 +32,7 @@
 #define IRremote_h
 
 #include <stdint.h>
+#include "IRremoteInt.h"
```

Comment:
As per other code review/PR. I'm not sure why, but it's error'ing on me if I use \<>'s. Otherwise I'd love too.